Logger/BaseLoggerMixins.cs

```diff
@@ -1,7 +1,44 @@
-﻿namespace Logger
+﻿using System;
+using System.Text;
+
+namespace Logger
 {
     public static class BaseLoggerMixins
     {
-        
+        public static void Error(this BaseLogger baseLogger, string message, params object[] argArray)
+        {
+            if (baseLogger is null)
+                throw new ArgumentNullException();
+
+            baseLogger.Log(LogLevel.Error, String.Format(message, argArray));
```

Comment:
From the coding guidelines

> DO use the C# keyword rather than the BCL name when specifying a data type (e.g., string rather than String). 

```suggestion
            baseLogger.Log(LogLevel.Error, string.Format(message, argArray));
```